python/dgl/contrib/dis_kvstore.py

```diff
@@ -473,6 +473,29 @@ def start(self):
                     data=res_tensor,
                     c_ptr=None)
                 _send_kv_msg(self._sender, back_msg, msg.rank)
+            # Init new data
+            elif msg.type == KVMsgType.NEW_DATA:
+                assert msg.rank == 0
```

Comment:
only one process sends the message?

Reply:
Yes. And other clients will just recv the message.